### PR TITLE
Creation of Posts

### PR DESCRIPTION
**Pull request by Hakdoff**

### CODE REVIEW

File: src/app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    
+    public function __construct()
+    {
+        $this->postCreate = new PostService;
+    }
+    
+    /**
+     * Create user posts
+     */
+    public function create(PostRequest $request)
```
Comment (hipe01mark) [inline src/app/Http/Controllers/PostController.php]: Return type

File: src/app/Models/User.php [hunk 2]
```diff
@@ -59,4 +59,12 @@ public function sendEmailVerificationNotification()
     {
         $this->notify(new VerifyEmail);
     }
+    
+    /**
+     * User has many posts
+     */
+    public function usersPosts()
```
Comment (hipe01mark) [inline src/app/Models/User.php]: return type

File: src/routes/web.php [hunk 3]
```diff
@@ -24,3 +26,7 @@
 
 // Dashboard Controller
 Route::get('/dashboard', [DashboardController::class, 'dashboard'])->middleware('auth');
+
+
+//Posts Routes
+Route::post('/post', [PostController::class, 'create']);
```
Comment (hipe01mark) [inline src/routes/web.php]: New line!

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by mikco-hipe — APPROVED** [review]

**Review by sunkenquest — APPROVED** [review]
LGTM 🙇🏼  pakiinvite ng account namin na hipe gmail para marecognize ang approval 🙇🏼

**Pull request merged**